src/main/resources/jenkins/plugins/logstash/LogstashJobProperty/config.jelly

```diff
@@ -1,5 +1,9 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
 	<f:optionalBlock name="enable" title="${descriptor.displayName}" checked="${instance != null}" help="/plugin/logstash/help/help.html">
+		<f:advanced>
+			<f:entry field="script" help="/plugin/logstash/help/help-script.html"/>
+			<f:property field="secureGroovyScript"/>
```

Comment:
field name doesn't match the variable name, this causes your problems with saving
secureGroovyScript <=> securedGroovyScipt

Reply:
@mwhudson , sorry but I don't see `securedGroovyScipt` used in the code anywhere. Also when field doesn't match class an exception is thrown. Or did I miss your point?

Reply:
you're right, must have had tomatoes on my eyes
Can you try to add an `inline=true` to the optionalBlock. Without it the nested stuff is not at the right place in the generated json

Reply:
```
<f:optionalBlock name="enable" title="${descriptor.displayName}" checked="${instance != null}" help="/plugin/logstash/help/help.html" inline="true">
```
Still didn't work :/